src/services/retry.service.ts

```diff
@@ -45,12 +45,15 @@ export class RetryService {
               this.logger.error('repoId or branchId is not provided');
               process.exit(1);
             }
-            const issues = await this.issuesService.getIssues(repoId, branchId);
-            if (issues.data.length === 0) {
+            const { issues, message } = await this.issuesService.getIssues(repoId, branchId);
+            if (issues.length === 0) {
               this.logger.log('No issues found');
               process.exit(0); // Exit with success
             } else {
-              await this.issuesService.drawIssuesTable(issues.data);
+              await this.issuesService.drawIssuesTable(issues);
+              if (message) {
+                console.log(message);
```

Comment:
Let's use `this.logger.log` here.

Reply:
replaced with `this.logger.log(message)`